service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -376,8 +278,11 @@ void shouldRemoveSupportingEvidenceDocumentWithCorrectID() {
                     .type(WITH_NOTICE)
                     .supportingEvidenceBundle(List.of(element(firstSupportingEvidenceID,
                         SupportingEvidenceBundle.builder()
-                        .document(supportingDocument)
-                        .build())))
+                            .document(supportingDocument)
+                            .build()), element(secondSupportingEvidenceID,
+                        SupportingEvidenceBundle.builder()
+                            .document(supportingDocument)
+                            .build())))
```

Comment:
this should be removed and instead at the end we should assert 

            assertThat(extractedCaseData.getAdditionalApplicationsBundle().get(2).getValue())
                .isEqualTo(expectedBundle);

as this is the bundle that should have the document removed.